website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -229,3 +229,71 @@ This endpoint supports pagination [with standard URL query parameters](/terrafor
   ]
 }
 ```
+
+## Show a Team Token
```

Comment:
Since we use "List" as the verb above, how would you feel about tweaking to match that language:
```suggestion
## List a team's token
```
Also, could you do me a huge favor and rewrite the headers in this file to use sentence case to match our style guide? 🙏

Reply:
Yes, can do on the sentence case!

Is  `List` the right verb here though? Generally in the API docs "List" means "show all the resources of this type" and "Show" means "show a single resource, with more detail". E.g. [Show and Organization](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organizations#show-an-organization)
We actually already have a "list" for this page https://developer.hashicorp.com/terraform/cloud-docs/api-docs/team-tokens#list-team-tokens